src/main/tls_config.c

```diff
@@ -155,15 +155,8 @@ static void _set_config_str_if_present(as_config *config, PyObject *tls_config,
 static char *get_string_from_string_like(PyObject *string_like)
 {
     char *ret_str = NULL;
-    PyObject *ustr = NULL;
-    if (PyString_Check(string_like)) {
-        ret_str = PyString_AsString(string_like);
-    }
-    else if (PyUnicode_Check(string_like)) {
-        ustr = PyUnicode_AsUTF8String(string_like);
```

Comment:
This was probably causing a reference count leak, `PyUnicode_AsUTF8String` returns a new reference.

Reply:
This shouldn't be a problem anymore because we removed the code right?

Reply:
yeah, shouldn't be an issue now